scss/form.scss

```diff
@@ -73,9 +75,21 @@ span.char-limit {
   }
 }
 
+.form-label {
```

Comment:
I don't see the font-size and line-height doing anything when I disable them in the inspector, and then it seems the margin gets overridden by the following selector. Can you double-check what this is all doing and make sure it's necessary? It feels like multiple ways to attack the same core elements.
<img width="1243" alt="image" src="https://user-images.githubusercontent.com/1682681/61145598-d749ae00-a4a5-11e9-8a12-de284a913c0a.png">